script/dex-65062-PDF_Resubmit.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+class Recreate686PDFProcessor
+  def initialize(claim_ids, report_file)
+    @report_file = report_file
+    @service = BGS::Services.new(external_uid: '1234', external_key: 'tbtest')
+    @claims =  SavedClaim::DependencyClaim.where(id: claim_ids)
+    headers = %w[created_at zip_uploaded_at id file_number url]
+    csv = headers.to_csv
+    file = report_file
+    File.write(file, csv)
+  end
+
+  def get_form_hash_686c(bgs_person, form)
+    {
+      'veteran_information' => {
+        'full_name' => {
+          'first' => bgs_person[:first_nm],
+          'middle' => '',
+          'last' => bgs_person[:last_nm]
+        },
+        'common_name' => bgs_person[:first_nm],
+        'va_profile_email' => bgs_person[:email_addr],
+        'email' => form['dependents_application']['veteran_contact_information']['email'],
+        'participant_id' => bgs_person[:ptcpnt_id],
+        'ssn' => bgs_person[:ssn_nbr],
+        'va_file_number' => bgs_person[:file_nbr],
+        'birth_date' => form['dependents_application']['veteran_information']['birth_date'],
+        'uuid' => '',
+        'icn' => ''
```

Comment:
I am assuming these are just here like this because the existing classes need this to not be null to not error, but they are hardcoded blank because you dont actually care/use them?

Reply:
copypasta, and then blanking out fields I didn't have and forgot to go back. I will test without them